internal/olm/manager.go

```diff
@@ -108,18 +113,22 @@ func (m *Manager) Status() error {
 	ctx, cancel := context.WithTimeout(context.Background(), m.Timeout)
 	defer cancel()
 
-	status, err := m.Client.GetStatus(ctx, m.Version)
+	version, err := m.Client.GetInstalledVersion(ctx)
+	if err != nil {
+		return err
+	}
+
+	status, err := m.Client.GetStatus(ctx, version)
 	if err != nil {
 		return err
 	}
 
-	log.Infof("Successfully got OLM status for version %q", m.Version)
+	log.Infof("Successfully got OLM status for version %s", version)
```

Comment:
Nit: any reason to change this to `%s` but leave the other "Successfully installed/uninstalled..." logs as `%q`?

Reply:
Nope, reverted.